modules/op-release-notes-1-10.adoc

```diff
@@ -152,4 +152,17 @@ $ oc get tektoninstallersets.operator.tekton.dev | awk '/pipeline-main-static/ {
 * This update adds a conversion configuration to the `ResolutionRequest` CRD. This update properly configures conversion from the `v1alpha1.ResolutionRequest` request to the `v1beta1.ResolutionRequest` request.
 * This update checks for duplicate workspaces associated with a pipeline task.
 * This update fixes the default value for enabling resolvers in the code.
-* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
\ No newline at end of file
+* This update fixes `TaskRef` and `PipelineRef` names conversion by using a resolver.
+
+[id="release-notes-1-10-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.1
+
+With this update, {pipelines-title} General Availability (GA) 1.10.1 is available on {product-title} 4.11 and later versions.
+
+[id="fixed-issues-1-10-1_{context}"]
+=== Fixed issues
+
+* Before this update, if the source branch information coming from payload included `refs/heads/` but the user-configured target branch only included the branch name, `main`, in a CEL expression, the push request would fail. With this update, {pac} passes the push request and triggers a pipeline if either the base branch or target branch has `refs/heads/` in the payload.
```

Comment:
maybe make a section that we are talking about pac, because that may confuse the user otherwise

Reply:
@chmouel, We have just 4 bullet points and the first 3 points clearly mentioned the association with PAC with ` With this update, {pac}.........`. IMO, this should not cause any ambiguity in readers' mind. Also, We already have this section for fixed issues and if i create a new one with the heading PAC, the heading level would go to 4 (====), which we might need to avoid. 

I can only change the heading to `=== Fixed issues for {pac}`, if we do not have other fixed issues for this release 1.10.1.